### DISCUSSION TITLE
3.8.0

### DISCUSSION THREAD

**Discussion by timmo001** [timmo001/system-bridge#2607] [General]
## What’s changed

You can now download a packaged version from the release of the application, installing python and using pip should no longer be nessasary.

> This is still experimental, only testing made has been for the windows setup file.

## ✨ New features

- Package with pyinstaller @timmo001 (#2576)
- Media controls @timmo001 (#2604)

## 🚀 Enhancements

- Media session changed @timmo001 (#2606)

## ⬆️ Dependency updates

- Bump @types/react-dom from 18.2.6 to 18.2.7 in /frontend @dependabot (#2569)
- Bump @types/node from 20.4.1 to 20.4.2 in /frontend @dependabot (#2570)
- Bump @types/react from 18.2.14 to 18.2.15 in /frontend @dependabot (#2571)
- Bump next from 13.4.9 to 13.4.10 in /frontend @dependabot (#2573)
- Bump eslint-config-next from 13.4.9 to 13.4.10 in /frontend @dependabot (#2574)
- Bump eslint from 8.44.0 to 8.45.0 in /frontend @dependabot (#2575)
- Bump clsx from 1.2.1 to 2.0.0 in /frontend @dependabot (#2577)
- Bump uvicorn[standard] from 0.22.0 to 0.23.0 in /backend @dependabot (#2578)
- Bump uvicorn[standard] from 0.23.0 to 0.23.1 in /backend @dependabot (#2579)
- Bump aiohttp from 3.8.4 to 3.8.5 in /gui @dependabot (#2581)
- Bump @mui/lab from 5.0.0-alpha.136 to 5.0.0-alpha.137 in /frontend @dependabot (#2583)
- Bump @mui/styles from 5.14.0 to 5.14.1 in /frontend @dependabot (#2582)
- Bump @mui/material from 5.14.0 to 5.14.1 in /frontend @dependabot (#2584)
- Bump aiohttp from 3.8.4 to 3.8.5 in /backend @dependabot (#2585)
- Bump aiohttp from 3.8.4 to 3.8.5 in /shared @dependabot (#2586)
- Bump eslint-config-next from 13.4.10 to 13.4.11 in /frontend @dependabot (#2587)
- Bump next from 13.4.10 to 13.4.11 in /frontend @dependabot (#2588)
- Bump zeroconf from 0.71.0 to 0.71.4 in /backend @dependabot (#2589)
- Bump eslint-config-next from 13.4.11 to 13.4.12 in /frontend @dependabot (#2590)
- Bump @types/node from 20.4.2 to 20.4.4 in /frontend @dependabot (#2592)
- Bump next from 13.4.11 to 13.4.12 in /frontend @dependabot (#2591)
- Bump @mui/material from 5.14.1 to 5.14.2 in /frontend @dependabot (#2594)
- Bump @types/react from 18.2.15 to 18.2.16 in /frontend @dependabot (#2595)
- Bump pyside6 from 6.5.1.1 to 6.5.2 in /gui @dependabot (#2596)
- Bump @types/node from 20.4.4 to 20.4.5 in /frontend @dependabot (#2597)
- Bump @types/react from 18.2.16 to 18.2.17 in /frontend @dependabot (#2598)
- Bump @types/lodash from 4.14.195 to 4.14.196 in /frontend @dependabot (#2599)
- Bump eslint from 8.45.0 to 8.46.0 in /frontend @dependabot (#2601)
- Bump fastapi from 0.100.0 to 0.100.1 in /backend @dependabot (#2602)

<hr /><em>This discussion was created from the release <a href='https://github.com/timmo001/system-bridge/releases/tag/3.8.0'>3.8.0</a>.</em>

**Comment by Daniel-dev22** [comment]
I am trying to set this up with docker but there's no instructions either here or the site except for the docker compose file with nothing in it in terms of configuration that was helpful.

How do I configure this for it to work? I entered the containers terminal and pulled the API key but home assistant is not accepting it. The only thing I see on the container logs is this which appears to me that nothing is happening. Attempting to pull any logs also fails.

```
s6-rc: info: service s6rc-oneshot-runner: starting
s6-rc: info: service s6rc-oneshot-runner successfully started
s6-rc: info: service fix-attrs: starting
s6-rc: info: service fix-attrs successfully started
s6-rc: info: service legacy-cont-init: starting
s6-rc: info: service legacy-cont-init successfully started
s6-rc: info: service legacy-services: starting
s6-rc: info: service legacy-services successfully started
```
Docker Compose
```yaml
version: "3.9"
services:
  system-bridge:
    container_name: system-bridge
    privileged: true
    restart: unless-stopped
    image: "ghcr.io/timmo001/system-bridge/amd64:3.8.2"
    volumes:
      - /home/dan/system-bridge:/data/system-bridge
    network_mode: host
    ports:
      - 9170:9170/tcp
```
